src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/TestEndpoints.cs

```diff
@@ -212,5 +260,9 @@ internal TestEndpoints(TestEndpoints testEndpoint, ConnectionString connString):
         // Graph Token Audience 
         public Uri GraphTokenAudienceUri { get; set; }
 
+        public Uri PublishingSettingsFileUri { get; set; }
+
+        //public Uri AADServiceUri { get; set; }
```

Comment:
We do not have any mapping in the Environment for this Key? Unless I am missing something that shows up in Environment that points to this Key?
All this was attempt to be align to what Get-AzureRmEnvironment shows, because if someone wants to setup a custome environment, they should be able to provide values for all these endpoint Uris